### ISSUE TITLE
Не реагирует на нажатие "Войти по токену" или "Войти по логину"

### ISSUE THREAD

**Issue by inv2004** [acherkashin/yandex-music-extension#91]
OS: Windows-10
Загружаю из WSL2

Кнопки нажимаются - но никакой реакции на это нету

**Comment by acherkashin** [comment]
@inv2004 На wsl я наверно и не тестировал. А просто под виндой всё работает?

**Comment by inv2004** [comment]
@acherkashin Да, ужалил всё, потом перезапустил vscode без wsl2, переставил и заработало. 

К сожалению без wsl2 (понятно что само расширение wsl2 не нужно) сам vscode не особо нужен, так так основная работа происходит именно в режиме связки с wsl2

**Comment by inv2004** [comment]
Как вариант на другой машине:

![image](https://user-images.githubusercontent.com/4949069/229494218-3f8d3513-94de-4173-9b1b-7d211ad12a63.png)

Ожидание:
  - работает без требования и установки в WSL

**Comment by lllzebralll** [comment]
popOS 22.04, code 1.77 тоже не работает, уже не первый раз пытаюс поставить, в феврале тоже неработало

**Comment by acherkashin** [comment]
А что в Output > Yandex Music Extension выводиться?

<img width="1455" alt="Screenshot 2023-04-04 at 07 53 55" src="https://user-images.githubusercontent.com/9947582/229690334-436ceb03-6d87-4e7c-a881-a04a9c8117fc.png">

**Comment by lllzebralll** [comment]
![image](https://user-images.githubusercontent.com/58988816/229718335-44642a9f-ef5c-43b6-8821-9dc86953b051.png)

**Comment by inv2004** [comment]
Если можно, я бы разделил issue на две. Так как тут обсуждается про запуск в linux, но, изначально это issue было про windows + WSL2
